kie-server-parent/kie-server-wars/kie-server/src/main/assembly/assembly-ee8-container.xml

```diff
@@ -0,0 +1,116 @@
+        <include>org.kie.server:kie-server-services-jbpm</include>
+        <include>org.kie.server:kie-server-services-jbpm-ui</include>
+        <include>org.kie.server:kie-server-services-case-mgmt</include>
+        <include>org.kie.server:kie-server-services-optaplanner</include>
+        <include>org.kie.server:kie-server-services-dmn</include>
+        <include>org.kie.server:kie-server-services-swagger</include>
+        <include>org.kie.server:kie-server-jms</include>
+        <include>org.kie.server:kie-server-rest-common</include>
+        <include>org.kie.server:kie-server-rest-drools</include>
+        <include>org.kie.server:kie-server-rest-jbpm</include>
+        <include>org.kie.server:kie-server-rest-jbpm-ui</include>
+        <include>org.kie.server:kie-server-rest-optaplanner</include>
+        <include>org.kie.server:kie-server-rest-dmn</include>
+        <include>org.kie.server:kie-server-rest-swagger</include>
+        <include>org.kie.server:kie-server-rest-case-mgmt</include>
+        <include>org.kie.server:kie-server-router-client</include>
+        <include>org.kie.server:kie-server-client</include>
+        <include>org.kie:jbpm-process-svg</include>
+        <include>org.kie:kie-security</include>
+        
+        <include>org.kie.server:kie-server-controller-websocket-client</include>
+        <include>org.kie.server:kie-server-controller-websocket-common</include>
+        
+        <include>org.kie.server:kie-server-services-openshift</include>
+
+        <include>org.jbpm:jbpm-services-ejb-timer</include>
+
+        <include>org.hibernate.common:hibernate-commons-annotations</include>
+        <include>org.hibernate:hibernate-entitymanager</include>
+        <include>org.hibernate:hibernate-core</include>
+        <include>org.hibernate:hibernate-validator</include>
+        <include>dom4j:dom4j</include>
+
+        <include>org.quartz-scheduler:quartz</include>
+
+        <include>com.google.inject.extensions:guice-servlet</include>
+        
+        <include>xerces:xercesImpl</include>
+      </includes>
+      <excludes>
+        <exclude>org.jboss.resteasy:*</exclude>
+        <exclude>org.jboss.spec.javax.annotation:*</exclude>
+        <exclude>org.jboss.spec.javax.ejb:*</exclude>
+        <exclude>org.jboss.spec.javax.interceptor:*</exclude>
+        <exclude>org.jboss.spec.javax.transaction:*</exclude>
+        <exclude>org.jboss.spec.javax.ws.rs:*</exclude>
+        <exclude>javax.enterprise:cdi-api</exclude>
+        <exclude>javax.inject:javax.inject</exclude>
+        <exclude>javax.validation:validation-api</exclude>
+        <exclude>stax:stax-api</exclude>
+        <exclude>javax.activation:activation</exclude>
+        <exclude>javax.persistence:javax.persistence-api</exclude>
+        <exclude>xml-apis:xml-apis:jar</exclude>
+        <exclude>javax.mail:mail</exclude>
+        <exclude>org.apache.cxf:cxf-rt-ws-policy</exclude>
```

Comment:
AFAIK this has been excluded due to compatibility issues with WebSphere. As ee8 classifier is used just by WildFly/EAP, would it have sense to comment this out (with description)?